test/.env.local

```diff
@@ -0,0 +1,9 @@
+ETHRPC=http://localhost:8545
+MYETH=0xfac5EC50BdfbB803f5cFc9BF0A0C2f52aDE5b6dd
+MYETHPK=0x06e48d48a55cc6843acb2c3c23431480ec42fca02683f4d8d3d471372e5317ee
+MYETH2=0x02fa1b44e2EF8436e6f35D5F56607769c658c225
+MYETH2PK=0x4faf826f3d3a5fa60103392446a72dea01145c6158c6dd29f6faab9ec9917a1b
+MYETH3=0xd8f468c1B719cc2d50eB1E3A55cFcb60e23758CD
+MYETH3PK=0x11f746395f0dd459eff05d1bc557b81c3f7ebb1338a8cc9d36966d0bb2dcea21
+CHAIN_ID=888
+UMEE_PEGGO_PATH=$PWD
```

Comment:
can we make this file as a not "hidden" (so without leading `.`)? It will be easier for everyone to see it.
Also, please update the README file and add note about the env file.